sdk/Lusid.Sdk/Client/Configuration.cs

```diff
@@ -51,12 +52,25 @@ public class Configuration : IReadableConfiguration
         public static readonly ExceptionFactory DefaultExceptionFactory = (methodName, response) =>
         {
             var status = (int)response.StatusCode;
+            
```

Comment:
is this a generated class/code?

Reply:
Yes this is indeed. We should be able to inject this functionality like Mike suggested in his comment https://finbourne.atlassian.net/browse/SENG-54